openmls/src/group/mls_group/membership.rs

```diff
@@ -245,18 +286,16 @@ impl MlsGroup {
             remove_proposal.clone(),
         )?);
 
-        self.plaintext_to_mls_message(remove_proposal, backend)
+        Ok(self.plaintext_to_mls_message(remove_proposal, backend)?)
     }
 
     /// Gets a list of [`KeyPackage`]s of the current group members.
-    pub fn members(&self) -> Result<Vec<&KeyPackage>, MlsGroupError> {
-        Ok(self
-            .group
-            .treesync()
-            .full_leaves()?
-            .iter()
-            .map(|(_, &kp)| kp)
-            .collect())
+    pub fn members(&self) -> Vec<&KeyPackage> {
+        match self.group.treesync().full_leaves() {
+            Ok(leaves) => leaves.iter().map(|(_, &kp)| kp).collect(),
+            // This should not happen, but this way we avoid returnung a library error
+            Err(_) => Vec::new(),
```

Comment:
If you really want to hide the library error here, can we add least trigger a debug log entry?